src/main/webapp/app/exercises/programming/shared/commits-info/commits-info.component.html

```diff
@@ -1,4 +1,8 @@
-<h5>{{ 'artemisApp.programmingExercise.commitInfo.title' | artemisTranslate }}</h5>
+@if (isRepositoryView) {
+    <h3>{{ 'artemisApp.programmingExercise.commitInfo.history' | artemisTranslate }}</h3>
+} @else {
+    <h5>{{ 'artemisApp.programmingExercise.commitInfo.title' | artemisTranslate }}</h5>
```

Comment:
Is it intentionally that you use different headlines here?

Reply:
Yes as the CommitInfo Component is used for different components but for the Commit History i prefer having Commit Hitsory instead of Commit Infos as Header